<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialEditor</class>
 <widget class="QWidget" name="MaterialEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>R</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="RealControl" name="colourRDoubleSpinBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>G</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="RealControl" name="colourGDoubleSpinBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="RealControl" name="colourBDoubleSpinBox"/>
   </item>
   <item row="5" column="1">
    <widget class="FileSelectWidget" name="textureFileSelectWidget"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Texture:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Colour</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="RealControl" name="roughnessDoubleSpinBox">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Roughness</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1">
    <widget class="RealControl" name="opacityDoubleSpinBox"/>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Opacity</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="RealControl" name="metallicFractionDoubleSpinBox"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Metallic Fraction</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="IndigoDoubleSpinBox" name="textureXScaleDoubleSpinBox">
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Texture x scale</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="IndigoDoubleSpinBox" name="textureYScaleDoubleSpinBox">
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Texture y scale</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealControl</class>
   <extends>QDoubleSpinBox</extends>
   <header>../qt/RealControl.h</header>
  </customwidget>
  <customwidget>
   <class>FileSelectWidget</class>
   <extends>QLineEdit</extends>
   <header>../qt/FileSelectWidget.h</header>
  </customwidget>
  <customwidget>
   <class>IndigoDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>../qt/IndigoDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
